lib/helpers/cli.js

```diff
@@ -287,15 +287,17 @@ export function getPossibleOptionNames(specifiedOptions) {
 }
 
 function executeGlobby(workingDir, pattern, ignore) {
-  let supportedExtensions = new Set(['.hbs', '.handlebars', ...GJS, ...SOON_DEPRECATED]);
```

Comment:
Why this change?

Reply:
I wanted to export as few things from the (now private) utility file as possible, so that they don't get misused (imported to a place that hadn't been intended).

Here, I thought duplication would be better. I kept `SOON_DEPRECATED`, though, because it seemed to have been created to indicate places where code may be later removed.